inst/include/forcing_component.hpp

```diff
@@ -15,17 +15,8 @@
  */
 
 #include "imodel_component.hpp"
-#include "ch4_component.hpp"
```

Comment:
👏 awesome, thank you

Reply:
> Do we want to remove the `comparison_data/` entirely? Would that information be useful to e.g. @dawnlwoodard 's `hectoreval`?

That is a great question for @dawnlwoodard if they are using it in `hectoreval` then we should leave it alone. Once upon a time CH had replacing the comparison data on my to-do list because we weren't entirely sure exactly where the data came from...